src/bindings/python/src/openvino/__init__.py

```diff
@@ -56,6 +56,24 @@
 from openvino._pyopenvino import RemoteTensor
 from openvino._pyopenvino import Op
 
+# Import opsets
+from openvino import opset1
+from openvino import opset2
+from openvino import opset3
+from openvino import opset4
+from openvino import opset5
+from openvino import opset6
+from openvino import opset7
+from openvino import opset8
+from openvino import opset9
+from openvino import opset10
+from openvino import opset11
+from openvino import opset12
+from openvino import opset13
+from openvino import opset14
+from openvino import opset15
+from openvino import opset16
```

Comment:
Could you please explain why those (all opsets) imports are needed here (and in `benchmark_tool/openvino/__init__.py`,  `ovc/openvino/__init__.py`)?

Reply:
It's needed if someone wants to use opsets like this:
```python
>>> import openvino as ov
>>> ov.opset15.constant([1,2,3])
```

Reply:
there is no other way to import those opsets collections dynamically, on demand?

Reply:
Init files for `openvino`, `benchmark_tool.openvino` and `ovc/openvino` have to be the same and it's enforced by cmake rule.

Reply:
Assuming that model inference is the main use case of openvino, I believe there is no need to import all of the opsets in openvino `__init__.py` file. A single opsetX can be imported by user as needed, without affecting the import time of the main openvino module (especially that the number of the opsets can grow).